extensions/cornerstone-dicom-rt/src/utils/initRTToolGroup.ts

```diff
@@ -0,0 +1,34 @@
+function createRTToolGroupAndAddTools(
+  ToolGroupService,
+  toolGroupId,
+  extensionManager
+) {
+  const utilityModule = extensionManager.getModuleEntry(
+    '@ohif/extension-cornerstone.utilityModule.tools'
+  );
+
+  const { toolNames, Enums } = utilityModule.exports;
+
+  const tools = {
+    active: [
```

Comment:
Could you export the default active tools from @ohif/extension-cornerstone, and then just apply the difference here?  Perhaps even better would be to get it from the customization service so that modes can set the default behaviour, then just modify the specifics here?